R/qenv-get_var.R

```diff
@@ -55,3 +55,29 @@ setMethod("[[", signature = c("qenv", "ANY"), function(x, i) {
     class = c("validation", "try-error", "simpleError")
   ))
 }
+
+#' @export
+`$.qenv.error` <- function(x, name) {
+  # Must allow access of elements in qenv.error object (message, call, trace, ...)
+  # Otherwise, it will enter an infinite recursion with the `conditionMessage(x)` call.
+  if (name %in% names(x)) {
+    return(NextMethod("$", x))
+  }
+
+  class(x) <- setdiff(class(x), "qenv.error")
```

Comment:
Safeguard to prevent infinite recursion, this can be "safely" removed if needed

Reply:
I need to think more about lines 66-70